python/src/Analytical_doc.i.in

```diff
@@ -83,8 +83,7 @@ estimate the failure probability, that is if:
 
 where :math:`\vect{x}^*` is the design point, :math:`s` is the threshold and :math:`\epsilon_t`
 is the absolute tolerance.
-Default value is specified in the `nearestPointAlgorithm` attribute. 
-
+The initial points are directly defined in the Normal Space through the used :class:`~openturns.OptimizationAlgorithm`.
```

Comment:
currently the optimization algorithm holds the starting point in the physical space, then FORM transforms it into the standard space before the design point search
so I would assume we should do the same for MultiStart

Reply:
Sorry, for the type, I typed not what I though...

Reply:
ah ok, but I guess we still want to transform MultiStart physical starting points into standard space in the code

Reply:
Not sure to understand : if we declare `optimAlgo`  as a multi start algorithm and then 

```python
algo = ot.FORM(optimAlgo, event)
```

Would it be working?

Reply:
OK understood. So I propose to fix it on the next Hackathon.